citesphere/src/main/webapp/WEB-INF/views/auth/group/item.html

```diff
@@ -31,34 +31,31 @@
 	});
 	
 	$(document).ready(function(){
-		$.ajax({ 
-			'url': '[(${citation.key})]/paging?index=[(${index})]&page=[(${page})]&sortBy=[(${sortBy})]&collectionId=[(${collectionId})]',
-			'type': "GET",
-			'success': function(response){
-				if(response.next != null){
-					document.getElementById("next").addEventListener("click", function(e) {
-						window.location.href = response.next + "?index=" + response.nextIndex + "&page=" + response.nextPage + "&sortBy=" + response.sortBy + "&collectionId=" + response.collectionId;
-					});
-					$('#next').attr("disabled", false);
-					$('#next').removeClass("btn btn-default").addClass("btn btn-primary");
+		var next = "[(${adjacentCitations.next})]";
+		if (next) {
+			$('#next').attr("disabled", false);
+			$('#next').removeClass("btn btn-default").addClass("btn btn-primary");
+			document.getElementById("next").addEventListener("click", function(e) {
+				uri = next + "?index=" + [(${adjacentCitations.nextIndex})] + "&page=" + [(${adjacentCitations.nextPage})] + "&sortBy=[(${sortBy})]&collectionId=[(${collectionId})]";
+				if ("[(${searchTerm})]") {
+					uri = uri + "&searchTerm=[(${searchTerm})]";
 				}
```

Comment:
Is there a reason this is not in an href attribute but handled in javascript?

Reply:
No particular reason. I'll change this.